app/controllers/additional_details/questionnaires_controller.rb

```diff
@@ -26,8 +26,7 @@ def create
   end
 
   def update
-    @questionnaire.update_attributes(questionnaire_params)
-    if @questionnaire.save
+    if @questionnaire.update(questionnaire_params)
```

Comment:
.update_attributes is an alias for .update as of 4.0.2
http://apidock.com/rails/v4.0.2/ActiveRecord/Persistence/update_attributes